pkg/volume/util/volumepathhandler/volume_path_handler.go

```diff
@@ -283,9 +284,6 @@ func (v VolumePathHandler) GetDeviceBindMountRefs(devPath string, mapPath string
 		return nil, err
 	}
 	for _, file := range files {
-		if file.Type()&os.ModeDevice != os.ModeDevice {
```

Comment:
Now that we know that this check is useless, can we add some other checks to reduce the side effects. For example, checking for common file types

Reply:
I think no. We expect this directory to be empty here. Anything inside this dir should result in failure, no matter what type it is, particularly if we replace the RemoveAll with rmdir, or rmdir will fail on non-empty directory.